### PR TITLE
Prod vs Int 

### PR DESCRIPTION
**Pull request by mariavkou**
make prodvsint

This script gives the differences between integration and production phase for:
-  ../1.0.0/WMTSCapabilities.xml
-  ../rest/services/all/MapServer/layersConfig
-  ../rest/services/ech/MapServer/layersConfig
-  ../rest/services/swissmaponline/MapServer/layersConfig
-  ../rest/services/api/MapServer
-  ../rest/services/all/faqlist
- +Catalog

### CODE REVIEW

File: scripts/prod_vs_int.sh [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+  exit
+fi
+echo "${background}${black}Current Git-status${reset}"
+git status
+echo "${background}${black}Git-adding the files${reset}"
+git add .
+echo "${background}${black}Current Git-status${reset}"
+git status
+echo "${background}${black}Git-commit at integration phase${reset}"
+git commit -m "Integration $now"
```
Comment (loicgasser) [inline scripts/prod_vs_int.sh]: can we possibly avoid an automatic commit/push in a productive app?

File: scripts/prod_vs_int.sh [hunk 2]
```diff
@@ -0,0 +1,121 @@
...
+cd ../db
+git checkout prod_vs_int_review
+
+# Now we are inside geoadmin/db and here the script is executable 
+# mkdir -p prod_vs_int_review 
+cd prod_vs_int_review
+
+# Date and time of executing the script
+now=$(date +%Y-%m-%d--%H:%M:%S)
+prodJsons="https://api3.geo.admin.ch/rest/services/all/MapServer/layersConfig|allLayersConfig.json
```
Comment (loicgasser) [inline scripts/prod_vs_int.sh]: can we get the base URLs direclty from `rc_int` ? Or at least define them once and reuse them later.

**Comment by loicgasser** [comment]
Nice and useful scritp thanks 2 comments otherwise LGTM

**Comment by gjn** [comment]
1. Use orphaned branch for the artefacts
2. Use temporary directory to handle git inteaction.

File: scripts/prod_vs_int.sh (hunk 1 above)
Comment (mariavkourouni) [inline scripts/prod_vs_int.sh]: Automatic commit is kept, but at the orphan branch now.

**Comment by loicgasser** [comment]
Can we close that?

**Comment by oterral** [comment]
@gjn ?

**Pull request closed without merging**